triggers/tests/src/test/scala/com/digitalasset/daml/lf/engine/trigger/simulation/CatTriggerResourceUsageTest.scala

```diff
@@ -7,13 +7,13 @@ package simulation
 import akka.actor.ActorSystem
 import akka.stream.Materializer
 import akka.stream.scaladsl.Sink
-import com.daml.ledger.api.refinements.ApiTypes.Party
 import com.daml.lf.data.Ref.QualifiedName
 import com.daml.lf.engine.trigger.test.AbstractTriggerTestWithCanton
 import com.daml.lf.speedy.SValue
 import org.scalatest.{Inside, TryValues}
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.wordspec.AsyncWordSpec
+import scalaz.syntax.tag._
```

Comment:
Changes in this file all relate to using `Party` rather than `String` as a method argument